pkg/network/transport/object/grpc/replication.go

```diff
@@ -206,6 +206,12 @@ func objectFromMessage(gMsg *objectGRPC.Object) (*object.Object, error) {
 }
 
 func (s *Server) metaInfoSignature(o object.Object) ([]byte, error) {
+	firstObj, _ := o.FirstID()
+	if firstObj.IsZero() && o.HasParent() && o.SplitID() == nil {
```

Comment:
last operand is inobvious to me. How about create a helper func for this? U check this condition twice here

Reply:
check is OK, i just suggest to share and doc it. Again, if i'd need to check whether object is first or not, i'd miss split ID check

Reply:
> i'd miss split ID check

as is said, this check is the only thing that differs v1 from v2. but if drop code is requested i usually support it, dropped